### ISSUE TITLE
Screen readers are not narrating label mapped information for the "Feedback title" edit field present in the dialog, while navigating using tab key

### ISSUE THREAD

**Issue by polatengin** [microsoft/vsts-extension-retrospectives#576]
Note: Same issue reproes throughout the application in all the dialogs

Other instances:
Other Instance 1: 
Same issue repro in 'Create new retrospective' dialog.
Path: Home -> Retrospective -> ... -> Create new retrospective.
Screen readers (NVDA/JAWS) are not narrating label mapped information for 'Title' dropdown, while navigating in focus & browse mode. Screen readers (NVDA/JAWS) should narrate label mapped information for 'Title' dropdown, while navigating in focus & browse mode.
Current Observation: NVDA is announcing as 'Please enter new retrospective name edit has autocomplete selected testing '.

Other instance 2: Collect tab
Path: Retrospective page--> Collect tab
Issue: Screen readers are not narrating grouping mapped information for the "Add new feedback buttons and feedback tiles" present in feedback section.
Ensure that: Screen readers should narrate as "What went well Add new feedback button".
PFA (1713304_Other Instance2_Screen readers are not narrating grouping mapped information for the Add new feedback buttons and feedback tiles.png)
Note: Same issue is observed wherever the similar items are present throughout the application.

Impact on a Person with Disability (PWD):
Visually challenged users who rely on assistive technologies and screen readers are impacted as Screen readers are not narrating label mapped information for the "Feedback title" edit field present in the dialog, while navigating using tab key

Test Environment:
OS: Windows 11 [Version 23H2 (OS Build 22631.2715)]
Browser: Edge Version (119.0.2151.97), Chrome Version (119.0.6045.200)
URL: [Work items - Boards (visualstudio.com)](https://worldwidelearning.visualstudio.com/MCAPS%20Accessibility/_workitems/myactivity/)
Screen Readers: NVDA 2023.1, JAWS(2022.2110.6)

Pre-requisites:
Enable NVDA/JAWS.

Repro Steps:
Step 1: Open MCAPS ADO in Edge/Chrome browser.
Step 2: Navigate to 'Retrospective' in left navigation and invoke it.
Step 3: Navigate to 'Groups' tab and invoke it --> more options(...) menu in feedback tiles--> Group feedback control --> Group feedback dialog.
Step 4: Navigate to the Edit field in the dialog using tab key
Step 5: Verify whether Screen readers are narrating label mapped information for the "Feedback title" edit field present in the dialog, while navigating using tab key or not.

Actual Result: 
Screen readers are not narrating label mapped information for the "Feedback title" edit field present in the dialog, while navigating using tab key
Current Observation: NVDA is announcing as "Enter feedback title edit".

Expected Result: 
Screen readers should narrate label mapped information for the "Feedback title" edit field present in the dialog, while navigating using tab key
Ex: NVDA should announce as "Search and select the feedback under which to group the current feedback Enter feedback title edit"

Suggestive Fixes:
1.Map 'id' attribute value of label tag to the 'for' attribute value of edit field.
2.Make the same change for remaining instances.

MAS Reference:
[MAS 3.3.2 – Labels or Instructions](https://microsoft.sharepoint.com/:w:/r/sites/accessibility/_layouts/15/Doc.aspx?sourcedoc=%7b3752B546-30D8-44D7-8221-6FC32EEB3AC8%7d&file=MAS%203.3.2%20%E2%80%93%20Labels%20or%20Instructions.docx&wdLOR=c36957864-4B45-43F1-AB1E-E2E3A9834EFC&action=default&mobileredirect=true)
 
WCAG Reference:
https://www.w3.org/WAI/WCAG21/Understanding/labels-or-instructions.html

AzDO Reference Link:
https://worldwidelearning.visualstudio.com/MCAPS%20Accessibility/_workitems/edit/1713304
